### PR TITLE
Enable RBAC after astro airflow init

### PR DESCRIPTION
**Pull request by andriisoldatenko**
tl;dr
fix https://github.com/astronomer/astronomer/issues/312

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
```
(airflow) ➜  demo_astro_cli astro-cli airflow create_user
airflow create_user -r Admin -u admin -e admin@example.com -f admin -l user -p admin
[2019-05-22 19:15:46,699] {settings.py:182} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=159
[2019-05-22 19:15:47,016] {__init__.py:51} INFO - Using executor LocalExecutor
[2019-05-22 19:15:47,725] {__init__.py:305} INFO - Filling up the DagBag from /usr/local/airflow/dags
[2019-05-22 19:15:48,173] {security.py:446} INFO - Start syncing user roles.
[2019-05-22 19:15:48,455] {security.py:357} INFO - Fetching a set of all permission, view_menu from FAB meta-table
[2019-05-22 19:15:48,649] {security.py:307} INFO - Cleaning faulty perms
admin already exist in the db
```

**Comment by andriisoldatenko** [comment]
```
(airflow) ➜  demo_astro_cli astro-cli airflow create_user
[airflow create_user -r Admin -u admin -e admin@example.com -f admin -l user -p admin]
[2019-05-22 19:01:45,133] {settings.py:182} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=116
[2019-05-22 19:01:45,444] {__init__.py:51} INFO - Using executor LocalExecutor
[2019-05-22 19:01:45,783] {manager.py:557} WARNING - No user yet created, use fabmanager command to do it.
[2019-05-22 19:01:46,092] {__init__.py:305} INFO - Filling up the DagBag from /usr/local/airflow/dags
[2019-05-22 19:01:46,463] {security.py:446} INFO - Start syncing user roles.
[2019-05-22 19:01:46,679] {security.py:357} INFO - Fetching a set of all permission, view_menu from FAB meta-table
[2019-05-22 19:01:46,862] {security.py:307} INFO - Cleaning faulty perms
Admin user admin created.
```

**Comment by schnie** [comment]
This is looking good! :)

I noticed the `create_user` command is an actual command under the `astro airflow` subcommand. Do you think it'd be possible to allow the user to run any `airflow` command via `astro airflow create_user ...`, `astro airflow backfill ...`? So rather than hard-coding the `create_user` function, we take anything after `astro airflow blah` and `exec` it as `airflow blah` inside the scheduler. This way we wouldn't have to pick and choose which commands to support.

I realize that we already use the `astro airflow` subcommand for other things like `init` and `deploy`, so maybe for now we nest it under `astro airflow run <airflow subcommand>`.

We may want to rearrange some of our command names, but not sure if we want to do that since people will already be used to them. cc @ryw @vparekh94 for any thoughts here.

**Comment by andriisoldatenko** [comment]
@schnie good point will refactor using `run`.

**Comment by schnie** [comment]
Looks great! Only thing I noticed was the error message when trying to `astro airflow run` when the local cluster isn't running:
```
$ astro airflow run version
Running: airflow version
Error: Error response from daemon: page not found
2019/05/24 22:17:17 Error response from daemon: page not found
```

Can we catch that earlier and show a more descriptive error message, like `Airflow is not running. Start it with 'astro airflow start'`?

**Comment by andriisoldatenko** [comment]
@schnie 
```
(airflow) ➜  demo_astro_cli astro-cli airflow run version
Running: airflow version
Error: airflow is not running, Start it with 'astro airflow start'
```

**Comment by andriisoldatenko** [comment]
@schnie please take another look when have free min ^

**Comment by schnie** [comment]
@andriisoldatenko nice, looks good. Think we should use this to automate a `create_user` with `admin/admin` for user pass? Can we just run it on start every time and swallow the error if it says user already created?

**Comment by andriisoldatenko** [comment]
@schnie do we need to show this output? or hide all logs too?
```
Env file ".env" found. Loading...
Sending build context to Docker daemon  12.29kB
Step 1/1 : FROM astronomerinc/ap-airflow:master-1.10.3-onbuild
# Executing 5 build triggers
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> 3f24536cb4eb
Successfully built 3f24536cb4eb
Successfully tagged demo-astro-cli_1d1c2c/airflow:latest
INFO[0000] [0/3] [postgres]: Starting
INFO[0001] [1/3] [postgres]: Started
INFO[0001] [1/3] [scheduler]: Starting
INFO[0001] Recreating scheduler
INFO[0001] [2/3] [scheduler]: Started
INFO[0001] [2/3] [webserver]: Starting
INFO[0001] Recreating webserver
INFO[0002] [3/3] [webserver]: Started
Airflow Webserver: http://localhost:8080/admin/
Postgres Database: localhost:5432/postgres
----------------->>>>>
Running: airflow create_user -r Admin -u admin -e admin@example.com -f admin -l user -p admin
[2019-05-28 11:37:23,398] {settings.py:182} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=27
[2019-05-28 11:37:23,772] {__init__.py:51} INFO - Using executor LocalExecutor
[2019-05-28 11:37:24,578] {__init__.py:305} INFO - Filling up the DagBag from /usr/local/airflow/dags
[2019-05-28 11:37:25,008] {security.py:446} INFO - Start syncing user roles.
[2019-05-28 11:37:25,320] {security.py:357} INFO - Fetching a set of all permission, view_menu from FAB meta-table
[2019-05-28 11:37:25,572] {security.py:307} INFO - Cleaning faulty perms
admin already exist in the db
<<<<<<<<<<<<<-----------------
```

**Comment by schnie** [comment]
@andriisoldatenko I think we can hide those logs from the initial create_user command.

**Comment by andriisoldatenko** [comment]
```
(airflow) ➜  demo_astro_cli astro-cli airflow start
Env file ".env" found. Loading...
Sending build context to Docker daemon  12.29kB
Step 1/1 : FROM astronomerinc/ap-airflow:master-1.10.3-onbuild
# Executing 5 build triggers
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> Using cache
 ---> 3f24536cb4eb
Successfully built 3f24536cb4eb
Successfully tagged demo-astro-cli_1d1c2c/airflow:latest
INFO[0000] [0/3] [postgres]: Starting
INFO[0001] [1/3] [postgres]: Started
INFO[0001] [1/3] [scheduler]: Starting
INFO[0002] [2/3] [scheduler]: Started
INFO[0002] [2/3] [webserver]: Starting
INFO[0003] [3/3] [webserver]: Started
Airflow Webserver: http://localhost:8080/admin/
Postgres Database: localhost:5432/postgres
(airflow) ➜  demo_astro_cli cd ../cd
```
done @schnie

**Comment by schnie** [comment]
Hmm, I just ran `astro airflow start` with the new CLI and it won't let me log in with admin/admin.

File: version/version.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func GetTagFromVersion(airflowVersion string) string {
 	version := CurrVersion
 
 	if !isValidVersion(version) || s.HasPrefix(version, "SNAPSHOT-") {
-		return fmt.Sprintf("master-onbuild-%s", airflowVersion)
+		return fmt.Sprintf("master-%s-onbuild", airflowVersion)
```
Comment (andriisoldatenko) [inline version/version.go]: @schnie fixed bug with Dockerfile during `astro airflow init`

**Pull request merged**